### PR TITLE
Update of contributing guide and related pages.

### PR DESCRIPTION
**Pull request by gwarf**
Update of contributing guide and related pages:
* style: split syntax example to a dedicated page

### CODE REVIEW

File: content/en/about/syntax/_index.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+```markdown
+{{%/* pageinfo */%}} This is a placeholder. {{%/* /pageinfo */%}}
+```
+
+Will render as:
+
+{{% pageinfo %}} This is a placeholder. {{% /pageinfo %}}
+
+#### General message
```
Comment (enolfc) [inline content/en/about/syntax/_index.md:64]: Duplicated

File: content/en/about/syntax/_index.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+```markdown
+{{%/* alert title="Note" color="info" */%}} This is a Note. {{%/* /alert */%}}
+```
+
+Will render as:
+
+{{% alert title="Note" color="info" %}} This is a Note. {{% /alert %}}
+
+#### Information message
```
Comment (enolfc) [inline content/en/about/syntax/_index.md:88]: Note and Info are the same?

**Review by enolfc — CHANGES_REQUESTED** [review]
Nice! Just 2 small comments

File: content/en/about/syntax/_index.md (hunk 1 above)
Comment (gwarf) [reply content/en/about/syntax/_index.md]: REmoved.

File: content/en/about/syntax/_index.md (hunk 2 above)
Comment (gwarf) [reply content/en/about/syntax/_index.md]: Indeed, duplicated too.

**Review by enolfc — APPROVED** [review]
👍

**Pull request merged**